js/packages/cli/src/cli.ts

```diff
@@ -583,4 +583,42 @@ program
 
 program.command('find-wallets').action(() => {});
 
+program
+  .command('sign_candy_machine_metadata')
+  .option(
+    '-e, --env <string>',
+    'Solana cluster env name',
+    'devnet', //mainnet-beta, testnet, devnet
+  )
+  .option(
+    '-k, --keypair <path>',
+    `Solana wallet location`,
+    '',
+  )
+  .option('-cndy, --candy-address <string>', 'Candy machine address', '')
```

Comment:
address may be taken from cache

Reply:
afaik address isn't being added to the .cache file only the config address.

i added a json file that will store all the candy machines you make from the cli inside the .cache folder